Documentation/network/concepts/masquerading.rst

```diff
@@ -72,7 +72,12 @@ The eBPF-based masquerading can masquerade packets of the following IPv4 L4 prot
 
 - TCP
 - UDP
-- ICMP (only Echo request and Echo reply)
+- ICMP
+
+.. note::
+
+    For ICMP, support is limited to Echo request, Echo reply, and the
+    error message Destination unreachable fragmentation needed.
```

Comment:
I think we often see “required” rather than “needed”? You mean [type 3 code 4](https://en.wikipedia.org/wiki/Internet_Control_Message_Protocol#Control_messages), right?

I'd also use quotes:

```suggestion
    error message "Destination unreachable, fragmentation required".
```

Reply:
Agree that it's more clear with your suggestion, thank you.